CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1861,8 +1867,14 @@ local function OnCoreGuiChanged(coreGuiType, coreGuiEnabled)
 			end
 		end
 		if showTopbarChatIcon then
-			if chatIcon then
-				AddItemInOrder(LeftMenubar, chatIcon, LEFT_ITEM_ORDER)
+			if Util.IsTouchDevice or ChatModule:IsBubbleChatOnly() then
+				if chatIcon and canChat then
+					AddItemInOrder(LeftMenubar, chatIcon, LEFT_ITEM_ORDER)
+				end
+			else
+				if chatIcon then
```

Comment:
Yeah, this is because there are 2 chat icons on mobile. We only want to get rid of the one that focus the chat bar, not the other one.